.github/workflows/comment_bot.yml

```diff
@@ -0,0 +1,23 @@
+name: comment_bot
+on:
+  issue_comment:
+    types: [created, edited]
+permissions:
+  contents: read
+
+jobs:
+  comment_bot:
+    if: startsWith(github.event.comment.body, '@bazel-io ')
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@91182cccc01eb5e619899d80e4e971d6181294a7 # v2.10.1
+        with:
+          egress-policy: audit
+
+      - name: Run bot
+        uses: bazelbuild/continuous-integration/actions/release-helper@728620684433bc54e4bb77613fa79977ba8da106 # master
```

Comment:
Feels weird we are mixing this with the release-helper, can we separate those?

Reply:
Done.